.goreleaser.yml

```diff
@@ -196,6 +202,20 @@ nfpms:
           mode: 0644
           owner: root
           group: root
+      - src: service/observiq-otel-collector
+        dst: /etc/init.d/observiq-otel-collector
+        type: config|noreplace
+        file_info:
+          mode: 0755
+          owner: root
+          group: root
+      - src: service/sysconfig/observiq-otel-collector
+        dst: /etc/sysconfig/observiq-otel-collector
```

Comment:
Can you review this page and decide if noreplace is correct here? https://www.cl.cam.ac.uk/~jw35/docs/rpm_config.html.

Trying to determine if we expect a user to make changes to this file. Maybe we do?